Multiplatform/Collections/Playlists/PlaylistAddSheet.swift

```diff
@@ -104,9 +105,24 @@ struct PlaylistAddSheet: View {
                 }
             }
             .navigationTitle("playlist.add.title")
+            #if targetEnvironment(macCatalyst)
+            .toolbar {
+                ToolbarItem(placement: .topBarTrailing) {
+                    Button {
+                        dismiss()
+                    } label: {
+                        Text("account.done")
+                    }
+                }
+            }
+            #endif
             .toolbarTitleDisplayMode(.inline)
         }
+        .foregroundStyle(colorScheme == .dark ? .white : .black)
+        // This indicator does nothing on macOS and will only confuse our user
+        #if !targetEnvironment(macCatalyst)
```

Comment:
It would be clearer if this was part of the previous `#if` statement using `#else`

Reply:
Is this really the case? This one is at the outer level of previous `#if` and cannot be used with `#else` directly as we cannot have unmatched braces in a `#if` branch.

Reply:
I did not know you have to add the `presentationDragIndicator` modifier to the first child, nvm